web/src/app/users/SendTestDialog.tsx

```diff
@@ -43,44 +43,42 @@ export default function SendTestDialog(
 ): JSX.Element {
   const { title = 'Test Delivery Status', onClose, messageID } = props
 
-  const [now] = useState(DateTime.local())
+  const [sendTestStatus, sendTest] = useMutation(mutation)
 
-  const [sendTest, sendTestStatus] = useMutation(mutation, {
+  const [{ data, fetching, error }] = useQuery({
+    query,
     variables: {
       id: messageID,
     },
+    requestPolicy: 'network-only',
   })
 
-  const { data, loading, error } = useQuery(query, {
-    variables: {
-      id: messageID,
-    },
-    fetchPolicy: 'network-only',
-  })
-
+  const now = useRef(DateTime.utc())
```

Comment:
Beings it's a constant value and has no need to cause a re-render I went with useRef. Should I use useState instead?

Reply:
In this case, they will be equivalent, but `useState` is more common/simple (well, at least in our codebase). If you don't call the set function, the state is stable.

These should work the same for this case:
```ts
const nowRef = useRef(DateTime.utc())
// console.log(nowRef.current)

const [now] = useState(DateTime.utc())
// console.log(now)
```

Since generally using refs requires careful usage, it signals: "hey something special may be happening here." Anything anywhere touching `ref.current` can change the value, and outside of the lifecycle funcs.

Using a state variable without a set function OTOH can be quickly scanned visually to know it's read-only and intentionally so.

---

In either case a `why` comment will settle it for any future reader. Something like `// We keep a stable timestamp to track how long the dialog has been open.`

Reply:
That makes sense. I'll switch it to useState and add the comment as well.